x-pack/plugins/reporting/export_types/common/execute_job/add_force_now_query_string.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+// @ts-ignore
+import url from 'url';
+import { ConditionalHeaders, KbnServer, ReportingJob } from '../../../types';
+import { getAbsoluteUrlFactory } from './get_absolute_url';
+
+function getSavedObjectAbsoluteUrl(job: ReportingJob, relativeUrl: string, server: KbnServer) {
+  const getAbsoluteUrl: any = getAbsoluteUrlFactory(server);
+
+  if (relativeUrl) {
+    const { pathname: path, hash, search } = url.parse(relativeUrl);
+    return getAbsoluteUrl({ basePath: job.basePath, path, hash, search });
+  }
+
+  throw new Error(`Unable to generate report. Url is not defined.`);
+}
+
+export const addForceNowQuerystring = async ({
+  job,
+  conditionalHeaders,
+  logo,
+  server,
+}: {
+  job: ReportingJob;
+  conditionalHeaders?: ConditionalHeaders;
+  logo?: any;
+  server: KbnServer;
+}) => {
+  // if no URLS then its from PNG which should only have one so put it in the array and process as PDF does
+  if (!job.urls) {
+    if (!job.relativeUrl) {
+      throw new Error(`Unable to generate report. Url is not defined.`);
+    }
```

Comment:
not sure if this block is needed since it looks like it happens on line 14 and the same error thrown on 19